<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Создание дела</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget {
    padding: 20px;
}

#planLabel {
    font-size: 50px;
}

QMainWindow {
	background-color: #fff
}

QPushButton[plan='true'] {
	background-color: #54ad54;
	border: none;
	border-radius: 0.25em;
	color: #fff;
	font-size: 20px;
}

QPushButton[small='true'] {
	background-color: #999;
	border: none;
	border-radius: 0.25em;
	color: #fff;
	font-size: 16px;
}

QPushButton[plan='true']:hover {
	background-color: #288b28;
}

QPushButton[small='true']:hover {
	background-color: #777;
}

QPushButton[plan='true']:pressed {
	background-color: #246024;
}

QPushButton[small='true']:pressed {
	background-color: #555;
}

#deleteBtn {
	border: none;/* 1px solid red;*/
/*	background-color: #F34235;*/
	background-color: #DC3545;
	border-radius: 0.25em;
	/*background-image: url(bin.png);*/
}

#deleteBtn:hover {
	background-color: #BB2D3B;
}

#saveBtn {
	background-color: #0D6EFD;
	border: none;
	border-radius: 0.25em;
	color: #fff;
	font-size: 18px;
}

#saveBtn:hover {
	background-color: #0B5ED7;
}

#saveBtn:pressed {
	background-color: #0B5ED7;
}

#titleInput, #categoryInput, #deadlineInput, #descriptionInput {
	border: 1px solid #ccc;
	border-bottom: 1px solid #54ad54;
	font-size: 15px;
}

QComboBox::down-arrow {
	image: url(images/drop-down-list.png);
}

QComboBox::down-arrow:on {
    image: url(images/drop-down-list-toggle.png);
}

QComboBox::drop-down {
	border-left: 1px solid #ccc;
}

QDateEdit::down-arrow {
	image: url(images/drop-down-list.png);
}

QDateEdit::up-arrow {
	image: url(images/drop-down-list-toggle.png);
}

QDateEdit::up-button {
	border-left: 1px solid #ccc;
	border-bottom: 1px solid #ccc;
}

QDateEdit::down-button {
	border-left: 1px solid #ccc;
}

QDateEdit:disabled {
	background-color: #eee;
}
/*
QDateEdit::up-button:hover {
	background-color: #ccc;
}

QDateEdit::up-button:pressed {
	background-color: #ccc;
}

QDateEdit::down-button:hover {
	background-color: #ccc;
}
*/
#titleLabel, #categoryLabel, #repeatLabel, #deadlineCheckBox, #descriptionLabel {
	font-size: 15px;
}

#deadlineCheckBox {
	background-color: #fff
}

#cancelBtn {
	background-color: #999;
	border: none;
	border-radius: 0.25em;
	color: #fff;
	font-size: 18px;
}

#cancelBtn:hover {
	background-color: #777;
}

#cancelBtn:pressed {
	background-color: #555;
}

#loadImageBtn {
	border: none;/* 1px solid red;*/
	background-color: #FFC107;
	border-radius: 0.25em;
	font-size: 18px;
}

#loadImageBtn:hover {
	background-color: #FFCA2C
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="windowLabel">
        <property name="styleSheet">
         <string notr="true">font-size: 20px</string>
        </property>
        <property name="text">
         <string>Создание дела</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="indent">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>Название:</string>
        </property>
        <property name="indent">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="titleInput"/>
      </item>
      <item>
       <widget class="QLabel" name="categoryLabel">
        <property name="text">
         <string>Категория:</string>
        </property>
        <property name="indent">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="categoryInput">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="repeatLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Повтор:</string>
        </property>
        <property name="indent">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="MondayBtn">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #0f0;
	border: 1px solid #ddd;</string>
          </property>
          <property name="text">
           <string>ПН</string>
          </property>
          <property name="isChecked" stdset="0">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">weekButtons</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="TuesdayBtn">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #0f0;
	border: 1px solid #ddd;</string>
          </property>
          <property name="text">
           <string>ВТ</string>
          </property>
          <property name="isChecked" stdset="0">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">weekButtons</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="WednesDay">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #0f0;
	border: 1px solid #ddd;</string>
          </property>
          <property name="text">
           <string>СР</string>
          </property>
          <property name="isChecked" stdset="0">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">weekButtons</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ThursdayBtn">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #0f0;
	border: 1px solid #ddd;</string>
          </property>
          <property name="text">
           <string>ЧТ</string>
          </property>
          <property name="isChecked" stdset="0">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">weekButtons</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="FridayBtn">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #0f0;
	border: 1px solid #ddd;</string>
          </property>
          <property name="text">
           <string>ПТ</string>
          </property>
          <property name="isChecked" stdset="0">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">weekButtons</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="SaturdayBtn">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #0f0;
	border: 1px solid #ddd;</string>
          </property>
          <property name="text">
           <string>СБ</string>
          </property>
          <property name="isChecked" stdset="0">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">weekButtons</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="SundayBtn">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #0f0;
	border: 1px solid #ddd;</string>
          </property>
          <property name="text">
           <string>ВС</string>
          </property>
          <property name="isChecked" stdset="0">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">weekButtons</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="deadlineFrame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="deadlineLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QCheckBox" name="deadlineCheckBox">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Дедлайн</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <property name="autoRepeat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="deadlineInput">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="descriptionLabel">
        <property name="text">
         <string>Описание:</string>
        </property>
        <property name="indent">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="descriptionInput"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="cancelBtn">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Отмена</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="loadImageBtn">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Загрузить картинку</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="saveBtn">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Добавить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="weekButtons"/>
 </buttongroups>
</ui>
